var/spack/repos/builtin/packages/ninja/package.py

```diff
@@ -74,6 +74,8 @@ def install(self, spec, prefix):
 
     def setup_dependent_package(self, module, dspec):
         name = "ninja"
+
         module.ninja = MakeExecutable(
-            which_string(name, path=[self.spec.prefix.bin], required=True), module.make_jobs
+            which_string(name, path=[self.spec.prefix.bin], required=True),
+            determine_number_of_jobs(self),
```

Comment:
The parameters for this according to https://spack.readthedocs.io/en/latest/spack.html?highlight=determine_number_of_jobs#spack.build_environment.determine_number_of_jobs are:

- parallel ([bool](https://docs.python.org/3/library/functions.html#bool) or None) – true when package supports parallel builds
- command_line ([int](https://docs.python.org/3/library/functions.html#int) or None) – command line override
- config_default ([int](https://docs.python.org/3/library/functions.html#int) or None) – config default number of jobs
- max_cpus ([int](https://docs.python.org/3/library/functions.html#int) or None) – maximum number of CPUs available. When None, this value is automatically determined.

Reply:
Quite right, it should have been `parallel=self.parallel`, change coming after running a different test since my first clearly didn't hit the right path.